enhancements/baremetal/bootstrap-kubelet-ip.md

```diff
@@ -0,0 +1,127 @@
+---
+title: bootstrap-kubelet-ip
+authors:
+  - "@tsorya"
+reviewers:
+  - "@hardys"
+  - "@dtantsur"
+  - "@dhellmann"
+  - "@patrickdillon"
+approvers:
+  - "@hardys"
+  - "@dtantsur"
+  - "@dhellmann"
+  - "@patrickdillon"
+api-approvers:
+  - None
+creation-date: 2022-03-23
+last-updated: 2022-03-23
+tracking-link:
+  - https://issues.redhat.com/browse/MGMT-11102
+see-also:
+  - https://bugzilla.redhat.com/show_bug.cgi?id=2070318
+replaces:
+superseded-by:
+---
+
+# Bootstrap External IP
+
+## Summary
+When installing a new cluster with UPI, you can set machine networks that will 
+be set to `networkConfig` field in the `install-config.yaml` file to configure the control plane network
+interfaces for the cluster hosts.  However, currently, you cannot configure 
+the bootstrap machine network using the same means.
+
+## Motivation
+
+In environments where bootstrap machine has more than one ip address, we should be able to set which ip that 
+will be configured as a hostIp in kubelet. This ip will be used in bootstrap kube-api service as advertised ip.
+The user can work around this by modifying the bootstrap ignition file; however,
+this isn't a friendly experience.
+
+### User Stories
+
+- As a user, I want to be able to specify host ip address for bootstrap machine
+
+### Goals
+
+- Support cluster installation in environments with bootstrap machine with more than one ip configured
+- Improve the level of customization of bootstrap networking
+- Improve user-friendliness when specifying a IP for the bootstrap
+
+### Non-Goals
+
+- Provide means for configuring every aspect of bootstrap networking
+
+## Proposal
+
+We will add a new fields, `bootstrapNodeIP` to allow for further customization of
```

Comment:
I think this should be a subnet and not a specific IP.

Reply:
then it will require to run nodeip-configuration, kubelet doesn't know what is subnet, it requires ip

Reply:
will add it as alternative